### PR TITLE
Robert/add other media

### PR DESCRIPTION
**Pull request by robmcl4**
Builds ontop of #123. Adds an 'Other Media' mutation / query, including tests.

OtherMedia is an Entry which has these unique properties:
* `path` - a path to a photo, non-null

### CODE REVIEW

**Comment by robmcl4** [comment]
Er... video submissions were added in a previous PR. We have three right now, "image", "video" and "other"

File: backend/db/migrations/20180121163800-create-other.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    path: {
+      type: Sequelize.STRING
+    },
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE
```
Comment (billdybas) [inline backend/db/migrations/20180121163800-create-other.js]: You should add `defaultValue: Sequelize.fn('NOW')` to this and `updatedAt`

File: backend/resolvers/mutations/entry.js [hunk 2]
```diff
@@ -44,3 +45,19 @@ export function createVideo (_, args, req) {
     }).then((entry) => Object.assign(entry, video.dataValues))
   })
 }
+
+export function createOtherMedia (_, args, req) {
+  if (req.auth.type !== ADMIN && !allowedToSubmit(args, req)) {
+    // don't allow non-admins to submit work claiming to be from someone else
+    throw new UserError('Permission Denied')
+  }
+  return Other.create({
```
Comment (billdybas) [inline backend/resolvers/mutations/entry.js]: Should probably be in a transaction.

**Comment by billdybas** [comment]
Also, probably need to rebase off of master.

**Pull request merged**